apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1136,7 +1136,8 @@ unsigned_tx_positive_test(Data, Params, HTTPCallFun, NewFun, Pubkey) ->
 tx_is_mined_test(AccountPubKey, TxHash) ->
     {ok, 200, #{<<"transactions">> := Txs}} =
         get_account_transactions(AccountPubKey, tx_encoding_param(json)),
-    true = lists:any(fun(#{<<"hash">> := H}) when H =:= TxHash -> true;
+    true = lists:any(fun(#{<<"block_hash">> := BH, <<"hash">> := H})
+                           when H =:= TxHash, BH =/= <<"none">> -> true;
```

Comment:
If `BH` binds to `<<"block_hash">>` unclear how it may be not `=/= <<"none">>`

Reply:
The signalling of the transaction being in mempool is to put `<<"none">>` there. A bit strange, but that is how the implementation is currently.